rskj-core/README.md

```diff
@@ -1,9 +1,15 @@
 
 ## rskj-core
 
+#### Configure the RSK-compatible Gradle local installation
+
+Run `sh configure.sh` in the project's root directory.
```

Comment:
Could you add Windows command? Ie, running `./configure.sh` from `git bash` in Windows 10. The `configure.sh` script is compatible with Windows

Reply:
Makes sense. Done!